### ISSUE TITLE
Discussion: include UCES in OpenRCT2 (or option to download and install) 

### ISSUE THREAD

**Issue by Patrik356b** [OpenRCT2/OpenRCT2#6941]
Suggested by a friend of mine.

I quite like the idea, but it's a pretty big download so I'm not sure.

What are your thoughts on this?

**Comment by spacek531** [comment]
The most I think OpenRCT2 should do, if at all, is offer a link to UCES. Bundling UCES with OpenRCT2 will increase bandwidth usage and can annoy people who want to enjoy OpenRCT2 without all those extra assets.

**Comment by janisozaur** [comment]
What is UCES?

**Comment by Lastorder-DC** [comment]
http://web.archive.org/web/20140706125041/http://www.rct2uces.com:80/disclaimer.html
UCES Scenarios/Objects/Rides cannot be redistributed

**Comment by duncanspumpkin** [comment]
I would put a link to it on the website.

**Comment by Gymnasiast** [comment]
Having a scenario tab would also be nice, but the scenario window can't fit more tabs, so that would need to be resolved first.

When the new object format comes around, having it list UCES as an origin for scenery items (besides RCT2, WW, TT) would also be nice.

I don't think we should _bundle_ it. I have a list of some high-quality scenery that actually matches the graphic style of the original and complements the available pieces nicely, and I watch that closely.

**Comment by rwjuk** [comment]
@Gymnasiast Is that list on the web anywhere?

**Comment by Gymnasiast** [comment]
@rwjuk Yes, it's on the wiki: https://github.com/OpenRCT2/OpenRCT2/wiki/Official-extended-scenery-set

**Comment by Patrik356b** [comment]
@Gymnasiast The RMC coaster is one of the few UCES rides that does have graphics that fits very well with the vanilla rides. (This is the only thing i actively use from UCES)

Regarding scenery and other things: it's very much hit-and-miss.

**Comment by spacek531** [comment]
Are you talking about X123M3-256's RMC? Because UCES was published in 2003, well before RMC got into ride design.

**Comment by Patrik356b** [comment]
@spacek531

Hmm. Seems odd, but the UCES may have been updated later. I'm pretty sure it came with UCES.

**Comment by spacek531** [comment]
@Patrik356b there's a lot of places you could have gotten it. If the thumbnail image has RMC track, it's definitely @X123M3-256's ride.

**Comment by Patrik356b** [comment]
@spacek531 I haven't downloaded rides nor parks with custom assets apart from UCES. (as far as i can tell)

**Comment by X123M3-256** [comment]
I think any scenery that is to be included with OpenRCT2 should closely match the vanilla assets. Very few objects would meet this standard (having tried to do this, it's really hard).

@Patrik356b 

>  I haven't downloaded separate rides nor parks with custom assets apart from UCES. (as far as i can tell)

If you've downloaded any parks  at all (or connected to a multiplayer server), are you sure the parks did not include custom assets? It's not that uncommon for an otherwise NCSO map to include a few custom trains.

 I think it's much more likely that you got the object from a park than UCES having been updated, since the original download page is long since dead and most of the people involved no longer play the game.
